Source/WebDriver/Session.h

```diff
@@ -159,7 +159,8 @@ public RefCounted<Session>
 #if ENABLE(WEBDRIVER_BIDI)
     void enableGlobalEvent(const String&);
     void disableGlobalEvent(const String&);
-    void dispatchEvent(RefPtr<JSON::Object>&&);
+    void dispatchBidiMessage(RefPtr<JSON::Object>&&);
+    void relayBidiCommand(const String&, unsigned commandId, Function<void(WebSocketMessageHandler::Message&&)>&&);
```

Comment:
The method doesn't need to know `commandId`, there is already 1-1 correlation between the command and its callback. The id is only needed for the websocket reply and could be added later by the caller.

Reply:
The issue here was that `relayBidiCommand` callback receives a `SessionHost::CommandResponse`, which is not used in `WebDriverService` (the caller). And as currently `WebSocketMessageHandler::Message` stores the messages as its String payload, 

I'll make `Message` store a regular JSON object, and we serialize it just when accessing the payload to send. That will make it easier for the caller to update individual message fields.

Reply:
Actually, I think it's better to make this `Message` change on a separate patch, as it might grow a bit.